gatsby/netlify.toml

```diff
@@ -2,5 +2,6 @@
   package = "netlify-plugin-gatsby-cache"
 
 [build]
+  ignore = "git diff —quiet HEAD^ HEAD . ../contents.json"
```

Comment:
@chmac this will ignore contents.json ?? can u confirm

Reply:
No, this will ignore any changes to the repo which are not `contents.json` or `.` which is the `gatsby/` directory.